backtester/datahandler/balanced_datahandler.py

```diff
@@ -0,0 +1,39 @@
+import pandas as pd
+from .datahandler import DataHandler
+from ..event import MarketEvent
+
+
+class BalancedDataHandler(DataHandler):
+    """Handler for balanced data set"""
+
+    def __init__(self, data_path, events):
+        data = pd.read_csv(data_path, parse_dates=["date"])
+
+        # We will assume bid and ask prices = close
+        data["bid"] = data["close"]
```

Comment:
I think that on another PR I have sen that you were correctly using bid and ask as two separate prices. Let's try to use the same criteria everywhere.

Reply:
Sorry my mistake, we don't have this information.